hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -53,20 +55,34 @@ name:
   uvm_test: otbn_base_test
   uvm_test_seq: otbn_base_vseq
 
-  // The directory where ELF binaries should be found.
-  //
-  // TODO: This needs deciding upon. Eventually, it might be nice to allow this
-  //       to be overridden on the command line ("I've got a bunch of ELF files
-  //       that stress my new feature; please run the test suite with them")
+  // A build mode that sets up the toolchain. Run meson via
+  // meson_init.sh to generate the .env file that describes our toolchain.
+  build_modes: [
+    {
+      name: otbn_build_env_mode
+      pre_build_cmds: ["cd {proj_root} && BUILD_ROOT={build_dir} ./meson_init.sh"]
```

Comment:
nit: Build modes are a collection of switches (`build_opts`, `run_opts`, `*_*_cmds`) that apply to a subset of tests. If ALL tests require those switches to be enabled, then a better solution is to move these switches outside of a build mode, 'bare' in the HJson. Creating a mode out of those is of course, a bit more cleaner, so I am fine either way.

Reply:
Good point. I'll ditch `otbn_build_env_mode`.